roles/openshift_aws/tasks/dns_route53.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: creating route53 zone(s)
+  route53_zone:
+    comment: "{{ openshift_aws_dns_zone }}"
+    state: present
+    vpc_id: "{{ ( l_openshift_aws_route53_scheme | ternary(vpcout.vpcs.0.id, '') ) }}"
+    vpc_region: "{{ openshift_aws_region }}"
+    zone: "{{ openshift_aws_dns_zone }}"
+  loop: "{{ openshift_aws_dns_records | selectattr('private_zone','defined') | map(attribute='private_zone') | list | unique  }}"
```

Comment:
To me, it appears there are exactly two possibilities for this, '' and vpcout.  Don't need to do some mapping, it's always only ever these two.  Might as well always create both of them, even if one never gets any entries.

If we add a zone key to the records, we could map that instead and with_nested: https://docs.ansible.com/ansible/2.4/playbooks_loops.html#nested-loops